src/components/Toc/Toc.scss

```diff
@@ -72,7 +72,7 @@ $leftOffset: 57px;
         scrollbar-width: thin;
 
         &_offset_top {
-            padding-top: 4px;
+            padding-top: 24px;
```

Comment:
Will it affect docs without toolbar?

Reply:
It doesn't depend on the toolbar. This will affect docs with `hideTocHeader`, there is padding instead of the toc title